packages/sdk/docs/mpl-token-auth-rules/classes/DataTypeMismatchError.html

```diff
@@ -25,7 +25,7 @@ <h4>Hierarchy</h4>
 <ul class="tsd-hierarchy">
 <li><span class="target">DataTypeMismatchError</span></li></ul></li></ul></section><aside class="tsd-sources">
 <ul>
-<li>Defined in <a href="https://github.com/metaplex-foundation/mpl-token-auth-rules/blob/7ab74a7/packages/sdk/src/generated/errors/index.ts#L40">packages/sdk/src/generated/errors/index.ts:40</a></li></ul></aside>
+<li>Defined in <a href="https://github.com/metaplex-foundation/token-authorization-rules/blob/57c4a96/packages/sdk/src/generated/errors/index.ts#L40">packages/sdk/src/generated/errors/index.ts:40</a></li></ul></aside>
```

Comment:
I still don't understand why your docs generate with the old repo name.  Maybe you still have the folder called `token-authorization-rules` locally?

Reply:
I do. Let me rename it.